src/renderer/controllers/playback-controller.js

```diff
@@ -269,6 +269,7 @@ module.exports = class PlaybackController {
     // update state
     state.playing.infoHash = infoHash
     state.playing.fileIndex = index
+    state.playing.fileName = fileSummary.name
```

Comment:
We should add a default value to the state file so that it acts as documentation. I will add that in a followup commit.